ang/test/civiawards/award-creation/directives/review-panels/review-panels.directive.spec.js

```diff
@@ -209,6 +209,61 @@
         });
       });
 
+      describe('when saving a review panel with missing groups', () => {
```

Comment:
The code was added inside `getFormattedVisibilitySettings` function. Which is called during loading the data, and not saving. So we need to test the `getFormattedVisibilitySettings` function and not `saveButtonClickHandler`.

So it should be something similar to `when editing an existing review panel` test suite.

Reply:
The method that filter the deleted groups (which is the new behaviour, the other is just to check the element of the array) is `getParamsForSavingReviewPanel`, that's why I tested the `saveButtonClickHandler`.